src/lib.rs

```diff
@@ -276,8 +276,7 @@ macro_rules! with_codes (($clens:expr, $max_bits:expr => $code_ty:ty, $cb:expr)
     //for bits in range_inclusive(1, $max_bits) {
     for bits in 1..$max_bits + 1 {
         code =
-            try!(
-                code.checked_add(bl_count[bits as usize - 1])
+            try!(code.checked_add(bl_count[bits as usize - 1])
```

Comment:
My point was the other way around, `code = try!(` on a line and this expression indented only once not twice.